scripts/update-tiledb-version.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
```

Comment:
just curious, why this file is here on this PR? seems like magic, but, old magic not new magic ...

Reply:
I though this and `.clang-format` were useful, and I used them to generate the code in this PR, so I threw them in...

Reply:
TIL 😁